molecule_inspec/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/verify.yml

```diff
@@ -2,86 +2,127 @@
-      set_fact:
-        inspec_download_url: "https://packages.chef.io/files/stable/inspec/3.9.3/ubuntu/14.04/inspec_3.9.3-1_amd64.deb"
-        inspec_download_shasha256: sha256:fe29d35fb95916b15310dc58787e63f22c4524be36a5c181307e08090cb0877d
-      when: (ansible_facts['os_family'] == "Debian" and (ansible_facts['distribution_major_version'] == "8" or ansible_facts['distribution_major_version'] == "14"))
-
-    - name: Setting variables (Debian 8 / Ubuntu 16.04)
-      set_fact:
-        inspec_download_url: "https://packages.chef.io/files/stable/inspec/3.9.3/ubuntu/16.04/inspec_3.9.3-1_amd64.deb"
-        inspec_download_shasha256: sha256:757dd2366a3932adc5fcc9382b30d77de6cc33152585f4c9f94f8918d9d349a7
-      when: (ansible_facts['os_family'] == "Debian" and (ansible_facts['distribution_major_version'] == "8" or ansible_facts['distribution_major_version'] == "16"))
-
-    - name: Setting variables (Debian 9 / Ubuntu 18.04)
-      set_fact:
-        inspec_download_url: "https://packages.chef.io/files/stable/inspec/3.9.3/ubuntu/18.04/inspec_3.9.3-1_amd64.deb"
-        inspec_download_shasha256: sha256:757dd2366a3932adc5fcc9382b30d77de6cc33152585f4c9f94f8918d9d349a7
-      when: (ansible_facts['os_family'] == "Debian" and (ansible_facts['distribution_major_version'] == "9" or ansible_facts['distribution_major_version'] == "18"))
-
-    - name: Download Inspec
-      get_url:
-        url: "{{ inspec_download_url }}"
-        dest: "{{ inspec_download_source_dir }}"
-        checksum: "{{ inspec_download_shasha256 }}"
-        mode: 0755
-
-    - name: Install Inspec (apt)
-      apt:
-        deb: "{{ inspec_download_source_dir }}/{{ inspec_download_url.split('/')[-1] }}"
-        state: present
-      when: ansible_pkg_mgr == "apt"
-
-    - name: Install Inspec (yum)
-      yum:
-        name: "{{ inspec_download_source_dir }}/{{ inspec_download_url.split('/')[-1] }}"
-        state: present
-      when: ansible_pkg_mgr == "yum"
-
-    - name: Create inspec test directory
-      file:
-        path: "{{ inspec_test_directory }}"
-        state: directory
-
-    - name: Copy inspec test directories
-      file:
-        path: "{{ inspec_test_directory }}/{{ item.path }}"
-        state: directory
-        mode: 0755
-      with_filetree: "{{ lookup('env', 'MOLECULE_VERIFIER_TEST_DIRECTORY') }}/"
-      when: item.state == 'directory'
-
-    - name: Copy inspec test files
-      copy:
-        src: "{{ item.src }}"
-        dest: "{{ inspec_test_directory }}/{{ item.path }}"
-        mode: 0644
-      with_filetree: "{{ lookup('env', 'MOLECULE_VERIFIER_TEST_DIRECTORY') }}/"
-      when: item.state == 'file'
-
-    - name: Execute Inspec tests
-      command: "{{ inspec_bin }} exec {{ inspec_test_directory }}"
-      register: test_results
-      ignore_errors: true
+    - name: Block | Non-Windows operating systems
+      block:
+        - name: Set facts/settings
+          set_fact:
+            inspec_bin: /opt/inspec/bin/inspec
```

Comment:
use https://omnitruck.chef.io/install.sh. This line can be changed to:

`inspec_bin: /usr/bin/inspec`